lib/tasks/redesign.rake

```diff
@@ -0,0 +1,12 @@
+namespace :css do
+  desc "Build your CSS bundle for the redesign"
+  task :redesign do
+    unless system "yarn install && yarn build:redesign"
+      raise "Command css:redesign failed, ensure yarn is installed and `yarn build:redesign` runs without errors"
+    end
```

Comment:
`system` is able to raise a more specific and verbose error if you pass `exception: true` to it.

https://rubyapi.org/3.1/o/kernel#method-i-system

```suggestion
  system "yarn install && yarn build:redesign", exception: true
```

Rails recently made a similar change to what I'm suggesting. https://github.com/rails/rails/commit/24fbbe06d214e587ece9e9292dcc9013f28d214f

Reply:
Good to know! Thanks for sharing.

I copy-pasted this from cssbundling-rails and jsbundling-rails. But for custom `system` calls I'll make sure to use `exception: true` in the future.